File: data/EXP01/golang_no_system_var/2527/2527.output_3.go
func xorBeauty(nums []int) int {
    n := len(nums)
    xorBeauty := 0
    for i := 0; i < n; i++ {
        for j := 0; j < n; j++ {
            for k := 0; k < n; k++ {
                effectiveValue := (nums[i] | nums[j]) & nums[k]
                xorBeauty ^= effectiveValue
            }
        }
    }
    return xorBeauty
}

File: data/EXP01/python/2527/2527.output_3.py
# @lc app=leetcode id=2527 lang=python3
class Solution:
    def xorBeauty(self, nums: List[int]) -> int:
        result = 0
        n = len(nums)
        
        for i in range(n):
            for j in range(n):
                for k in range(n):
                    effective_value = (nums[i] | nums[j]) & nums[k]
                    result ^= effective_value
        
        return result